scripts/register_host.sh

```diff
@@ -15,7 +15,9 @@ printf "\nCreating host zone...\n"
 $MAIN_STRIDE_CMD tx stakeibc register-host-zone \
     connection-0 $ATOM_DENOM cosmos $IBC_ATOM_DENOM channel-0 1 \
     --from $STRIDE_ADMIN_ACCT --gas 1000000 --home $SCRIPT_DIR/state/stride1 -y
-CSLEEP 10
+
+# sleep a while longer to wait for ICA accounts to set up
+sleep 60
```

Comment:
nit^2: why did we move from CSLEEP to sleep? Was kind of nice to see the countdown

Reply:
I think that's outdated (sorry I think the rebasing made these diffs a little wonky). I removed the CSLEEPS cause we now never sleep more than a few seconds